tests/e2e/nextcloud/00-assert.yaml

```diff
@@ -20,13 +20,9 @@ spec:
         instances: 1
       relativePath: /
       serviceLevel: besteffort
-      version: "23"
-      #version: 23.0.5-202402021353-44-af6cea11
+      version: "29"
```

Comment:
Do we even need to test the version? Will have to update it all the time. If we can omit the version from the claim we  can remove version from testing.

Reply:
Fine by me